lms/static/styles/components/_Dialog.scss

```diff
@@ -68,8 +69,7 @@
   justify-content: flex-end;
   margin-top: 10px;
 
-  .Button {
-    @include focus.outline-on-keyboard-focus;
+  * {
```

Comment:
Can you help me understand this universal selector here? It is making me nervous :)